package.json

```diff
@@ -39,28 +39,16 @@
     "start": "nodemon --watch src --exec npm run build -e ts,tsx"
   },
   "dependencies": {
-    "react-select": "5.8.0"
+    "react-select": "5.8.x"
```

Comment:
I realized I don't need to be so strict as to not allow other patch versions of `react-select`, so this just opens it up for the others.